### PR TITLE
Lovely readme

### PR DESCRIPTION
**Pull request by wemmm**
And the cherry on top: a lovely readme for team MYTM!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,64 @@
...
+$$$$$$$$"$$$c$$$cc$$$c $$$""      $$$""    
+888 Y88" 888o888   888,888o       888o     
+MMM  M'  "MMMYMM   ""` YMMMb      YMMMb   
+
+   an interactive mind mapping tool
+```
+
+## What is this cool thing?
+
+An interactive mind map! We are using Node.js, Express, Javascript, JQuery, Pug and HTML5 Canvas to make a board that you can add ideas to, then link your ideas to other ideas. Possibly with a snazzy interface.
```
Comment (charlottebrf) [inline README.md]: Could we replace Javascript with JavaScript ES6? 💯

File: README.md [hunk 2]
```diff
@@ -1,6 +1,64 @@
...
+
+We're testing with Mocha and Zombie and potentially Chai.
```
Comment (charlottebrf) [inline README.md]: Could we change this to: "We're testing with Mocha, Chai, Supertest and Zombie."

File: README.md [hunk 3]
```diff
@@ -1,6 +1,64 @@
...
+
+## How do I get in on this?
+
+Not deployed to web yet- watch this space! We may also potentially deploy via Electron.
```
Comment (charlottebrf) [inline README.md]: I vote we add this to Heroku & also include some instructions for running this on a server/ installing dependencies. I'm happy to add in after we merge first version?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,64 @@
...
+
+## How do I use it?
+
+![mapp](https://github.com/charlottebrf/mytm/blob/lovely-readme/images/mindmapp.png)
+
+The green bar on the left has a selection of tools to help you map out your ideas- you can type a word in the text box, then click the shape that you would like to create. The idea will then appear on the canvas to the right, and can be clicked and dragged around with the mouse. You can then add lines between ideas by double clicking a start point and an end point.
+
+You can clear lines but leave ideas on the canvas by clicking the 'Clear Lines' button on the toolbar, or both lines and ideas by clicking 'Clear Canvas'. Different canvas colours can be toggled with the row of circular coloured buttons, and the toolbar can be shown or hidden with the toggle in the lower left.
```
Comment (charlottebrf) [inline README.md:43]: Great instructions thanks @wemmm  :)

File: README.md [hunk 5]
```diff
@@ -1,6 +1,64 @@
...
+
+An instruction sheet can be toggled with the '?' button.
+
+## Our Approach
+
+We spent a good amount of time at the beginning of the project discussing user stories and the most appropriate technologies to use to accomplish them. We ultimately settled on using Node.js for the server and backend, and HTML5 Canvas for the mind map itself- meaning we would be using JavaScript to interface with it and render lines and shapes. The user interface was made with CSS3 and a very small amount of JavaScript for toggling parts of the display.
```
Comment (charlottebrf) [inline README.md]: Can we change JavaScript to JavaScript ES6 again?

File: README.md [hunk 6]
```diff
@@ -1,6 +1,64 @@
...
+Initial diagramming:
+
+
+![Diagramming](https://github.com/charlottebrf/mytm/blob/lovely-readme/images/diagramming.jpg)
+
+
+![Diagramming](https://github.com/charlottebrf/mytm/blob/lovely-readme/images/diagramming2.jpg)
+
+
+We encountered a lot of entertaining glitches with our shapes- especially when trying to get them to display text:
```
Comment (charlottebrf) [inline README.md:61]: hehe

File: README.md [hunk 7]
```diff
@@ -38,7 +96,21 @@ So that I can return to my beautifully organised ideas,
 I would like to save and load a mind map
 ```
 
-# TEAM MAP YOUR TECH MIND CHARTER
+## Who is Team MYTM?
+
+The following cool cats:
```
Comment (charlottebrf) [inline README.md:107]: 💯

File: README.md [hunk 8]
```diff
@@ -52,13 +124,23 @@ Wellbeing comes first
 We are not afraid of new technologies
```
Comment (charlottebrf) [inline README.md:130]: Nice!

File: README.md [hunk 9]
```diff
@@ -52,13 +124,23 @@ Wellbeing comes first
...
 - Improve front end testing abilities
 - Improve understanding of Javascript
 - Use multiple technologies on a single project
 - Strong team communication with [waffle](https://waffle.io/charlottebrf/mytm) and [slack](https://slack.com/)
 - Use Agile & XP values
-```
+
+
+## Resources and Tutorials That We Have Deeply Appreciated
+
```
Comment (charlottebrf) [inline README.md:145]: Great addition

File: README.md [hunk 10]
```diff
@@ -1,6 +1,64 @@
...
+.        :    :::.  ::::::::::.::::::::::.
+;;,.    ;;;   ;;`;;  `;;;```.;;;`;;;```.;;;
+[[[[, ,[[[[, ,[[ '[[, `]]nnn]]'  `]]nnn]]'
+$$$$$$$$"$$$c$$$cc$$$c $$$""      $$$""    
+888 Y88" 888o888   888,888o       888o     
+MMM  M'  "MMMYMM   ""` YMMMb      YMMMb   
+
+   an interactive mind mapping tool
+```
```
Comment (charlottebrf) [inline README.md:19]: I'm going to see if I can add Travis CI to this project so we can have a Travis badge also

**Review by charlottebrf — CHANGES_REQUESTED** [review]
Amazing job @wemmm a few comments from my end & some things that I'm happy to add in a bit later on also

File: README.md (hunk 3 above)
Comment (wemmm) [reply README.md]: We should definitely deploy! I'll add local instructions for now, then we can add a link when we have one.

File: README.md (hunk 5 above)
Comment (wemmm) [reply README.md]: Yep yep! I'll leave the latter JavaScript as is though because my UI JS isn't actually in ES6 sadly.

**Review by charlottebrf — APPROVED** [review]
:shipit:  real good!

**Pull request merged**